recipes/openfx/all/CMakeLists.txt

```diff
@@ -6,18 +6,17 @@ add_definitions(-D_HAS_AUTO_PTR_ETC)
 
 # Flags
 if(MSVC)
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DWINDOWS -DNOMINMAX")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DWINDOWS -DNOMINMAX -D_WIN32")
 else()
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wno-deprecated -Wno-deprecated-declarations")
+	add_definitions(-Dlinux)
 endif()
 
 # Conan
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup(KEEP_RPATHS)
-find_package(EXPAT REQUIRED)
+find_package(expat REQUIRED)
```

Comment:
If this is also depending on opengl shouldn't be a find_package for opengl here?